decidim-core/app/cells/decidim/card_cell.rb

```diff
@@ -28,11 +28,11 @@ def resource_cell
     end
 
     def title
-      model.try(:title) || model.try(:name) || ""
+      decidim_escape_translated(model.try(:title) || model.try(:name) || "")
```

Comment:
In its equivalent in `develop` we're not doing the escape, so I'd prefer to be  consistent:

https://github.com/decidim/decidim/blob/3dab60cb3a2ec643fd998e19711aadf067f83eb7/decidim-core/app/cells/decidim/card_cell.rb#L60C27-L60C28

If we want to do it in v0.27 then we should do the change in develop and then backport it

Reply:
reverted in [bde61ef](https://github.com/decidim/decidim/pull/12547/commits/bde61efea5e7275a335ef994faa30ba21f9aff15)

Reply:
If there's a good reason then we can leave it as is

Reply:
We need to leave it, as the tests are failing otherwise.